package.json

```diff
@@ -12,6 +12,7 @@
     "e2e:publish": "./scripts/e2e-publish.sh",
     "e2e:resolutions": "node ./scripts/e2e-resolutions.js",
     "examples": "npm run examples --workspaces --if-present",
+    "examples:build": "npm run examples:build --workspaces --if-present",
```

Comment:
Have kept this since I already had this in and this will be useful, but then I decided that I did not (yet) want to build the examples (which I initially had) - since that would confuse users with the new constructors and all - and so I force-pushed again removing this again.